blog/index.html

```diff
@@ -88,6 +88,8 @@ <h5 class="card-header"><i class="fa fa-tags"></i> Tags</h5>
           <div class="carousel-item {% if forloop.first %}active{% endif %}">
             <img class="card-img-top mb-2" src="{{ s.cover-image }}" alt="spotlight image of {{ s.title }}">
             <div class="card-body">
+              <h4 class="card-title">{{ s.status }} Spotlight</h4>
```

Comment:
Should set default if status is not present.

Reply:
As a note, this card is used to template _each_ card in that carousel, not whats above or under it. So the "Alumni Spotlight" text doesn't make that much sense. I think putting a `<h6 class="card-subtitle text-muted">{{ s.content | default: "Alumnus" }}</h6>` between line 94 and line 95 would look better.